pkg/cnab/provider/action.go

```diff
@@ -232,6 +243,10 @@ func (r *Runtime) SaveOperationResult(opResult driver.OperationResult, installat
 
 	for outputName, outputValue := range opResult.Outputs {
 		output := result.NewOutput(outputName, []byte(outputValue))
+		output, err = r.sanitizer.CleanOutput(output, cnab.ExtendedBundle{run.Bundle})
+		if err != nil {
+			bigerr = multierror.Append(bigerr, errors.Wrapf(err, "error sanitizing sensitive %s output for %s run of installation %s\n%#v", output.Name, run.Action, installation, output))
```

Comment:
To avoid leaking sensitive data, let's not print the output value as well, just the other fields.

Reply:
I believe the value field of the output after calling `CleanOutput()` will always be cleared out. It shouldn't have leakage issue

Reply:
Good point! I changed the `CleanOutput` to always clear out the value field when there's an error

Reply:
I really like that 👍 It should help prevent us from accidentally persisting it if the clean failed.